talestation_modules/code/modules/antagonists/bloodsuckers/bloodsucker/bloodsucker_life.dm

```diff
@@ -0,0 +1,429 @@
+/// Runs from COMSIG_LIVING_LIFE, handles Bloodsucker constant proccesses.
+/datum/antagonist/bloodsucker/proc/LifeTick()
```

Comment:
PascalCase

Reply:
this is an important proc so I want it to stand out.